android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -374,7 +377,18 @@ public void onPropertyChanged(String name, Object value)
 				}
 			}
 		}
-
+		if (name.equals(TiC.PROPERTY_BAR_COLOR)) {
+			// Get a reference to the ActionBar.
+			ActionBar actionBar = ((AppCompatActivity) activity.get()).getSupportActionBar();
```

Comment:
We need to do a `null` check on the activity reference in case the property was changed after the activity window was closed/destroyed.

Also, the above code uses the `windowActivity` member variable instead. Would you mind switching to that for consistency please? I can see that `windowActivity` is assigned via `windowCreated()`. So, we know it's never assigned to the previous activity.